Sources/RemoteMessagingTestsUtils/MockRemoteMessagingStore.swift

```diff
@@ -66,11 +66,6 @@ public class MockRemoteMessagingStore: RemoteMessagingStoring {
         return scheduledRemoteMessage
     }
 
-    public func fetchRemoteMessage(withID id: String) -> RemoteMessageModel? {
-        fetchRemoteMessageCalls += 1
```

Comment:
I think the variables `fetchRemoteMessageCalls` and `hasDismissedRemoteMessageCalls` can be deleted too 🙂

Reply:
Right, let me delete them!